app/uploaders/response_document_uploader.rb

```diff
@@ -1,4 +1,4 @@
-class ConsultationResponseFormUploader < WhitehallUploader
+class ResponseDocumentUploader < WhitehallUploader
```

Comment:
Is this right that we've removed the "Consultation" element from this? I suppose the uploader can easily be shared?

Reply:
Yes we have! In contrast to the other renaming, this one actually has nothing tied to Consultation in the code. So we thought we could rename it to something generic between the two and reuse for any type of Response Document (judging by the file extension allowlist)